srcpkgs/removed-packages/template

```diff
@@ -90,9 +91,9 @@ replaces="
  fuse-usmb<=20130204_7
  gandi-cli<=1.6_2
  gconf-editor<=3.0.1_3
- gconfmm<=2.28.3_4
  gconfmm-devel<=2.28.3_4
  gconfmm-doc<=2.28.3_4
+ gconfmm<=2.28.3_4
```

Comment:
Most of these are from me. How did you sort them? I ran :sort in vim and it gave me gconfmm before gconfmm-devel.

Reply:
I quickly hacked up this contraption, but it also just ran `:sort` in vim. Might me locale difference?
```bash
xrmpkg() {
	local toremove=$1
	local toremovever=$(xbps-uhelper getpkgversion $(xbps-query -Rp pkgver "${toremove}")) # 1.2_3
	local subpkgs=$(xsubpkg "${toremove}" | xargs -I{} echo "srcpkgs/{}")
	
	# remove package and subpkg symlinks
	git rm -r srcpkgs/"${toremove}" ${subpkgs}
	
	# add the pkg<=version string to removed-packages
	vim -es "+%s/replaces=\"/replaces=\"\r ${toremove}<=${toremovever}" '+wq' srcpkgs/removed-packages/template
	vim -es '+0/replaces=/+1,\//\"/sort' '+wq' srcpkgs/removed-packages/template
	vim -es '+0/replaces=/+1,\//\"/uniq' '+wq' srcpkgs/removed-packages/template
	
	# remove any mentions from shlibs, this might match too much
	vim -es "+%g/${toremove}/d" '+wq' common/shlibs
	
	# prepare commit, check changes
	git commit -e -v -m "${toremove}: remove package" -- srcpkgs/"${toremove}" ${subpkgs} srcpkgs/removed-packages/template common/shlibs
}
```